pex/variables.py

```diff
@@ -93,6 +95,25 @@ def strip_default(self, instance):
         except NoValueError:
             return None
 
+    def value_or(
+        self,
+        instance,  # type: _O
+        fallback,  # type: _P
+    ):
+        # type: (...) -> _P
+        """Return the value of this property without the default value applied or else the fallback.
```

Comment:
To confirm, this is _without_ the default? That's not a typo?

If so, perhaps:

> Return the value of this property, without the default value applied, and falling back to `fallback`.

Reply:
Confirmed. I do not see how the new words make things more clear.